src/OmniSharp.Roslyn/XmlDocumentationProvider.cs

```diff
@@ -1,55 +0,0 @@
-﻿using System;
```

Comment:
Note: This file is almost identical (at least, the "meat" of it is identical) to https://github.com/dotnet/roslyn/blob/version-2.0.0-rc/src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs, which we can now use.